File: Vidly/Scripts/Customs/Classes/QueryObject.js

export class QueryObject {
    constructor(page, pageSize, search, searchBy, comparison, orderBy, sortBy) {
        this.page = page;
        this.pageSize = pageSize;
        this.searchBy = searchBy;
        this.search = search;
        this.comparison = comparison;
        this.orderBy = orderBy;
        this.sortBy = sortBy;
    }
}

File: Vidly/Customs/Models/QueryObject.cs
namespace Vidly.Customs.Models
{
  public class QueryObject
  {
    public string SortBy { get; set; } = "asc";
    public string OrderBy { get; set; } = "Id";
    public int Page { get; set; } = 1;
    public int PageSize { get; set; } = 10;
    public string SearchBy { get; set; } = "";
    public string Search { get; set; } = "";
    public string Comparison { get; set; } = "contains";
    public string Fields { get; set; } = "";
  }

}